content/cloud-docs/api-docs/providers.mdx

```diff
@@ -172,6 +173,9 @@ curl \
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
+#### Private providers
```

Comment:
Might be worth a sentence like 181, that when a private provider is created, it is only usable by organization members?

Reply:
good call 👍